components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -793,9 +834,13 @@ func (r *Reconciler) stopNATSController() error {
 	return nil
 }
 
-func (r *Reconciler) startBEBController(clientID, clientSecret []byte) error {
+func (r *Reconciler) startBEBController(clientID, clientSecret, tokenURL []byte) error {
 	if !r.bebSubMgrStarted {
-		bebSubMgrParams := subscriptionmanager.Params{"client_id": clientID, "client_secret": clientSecret}
+		bebSubMgrParams := subscriptionmanager.Params{
+			subscriptionmanager.ParmaNameClientID:     clientID,
+			subscriptionmanager.ParmaNameClientSecret: clientSecret,
+			subscriptionmanager.ParmaNameTokenURL:     tokenURL,
+		}
 		if err := r.bebSubMgr.Start(r.cfg.DefaultSubscriptionConfig, bebSubMgrParams); err != nil {
```

Comment:
hope `bebSubMgrParams` are not printed anywhere

Reply:
I did not introduce any logs that print it.